cpp/CMakeLists.txt

```diff
@@ -16,10 +16,13 @@
 
 cmake_minimum_required (VERSION 3.11)
 
+project (libphonenumber VERSION 8.12.0)
+
 # Pick the C++ standard to compile with.
 # Abseil currently supports C++11, C++14, and C++17.
 set(CMAKE_CXX_STANDARD 11 CACHE STRING "C++ standard used to compile this project")
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 
 project (libphonenumber)
```

Comment:
Please remove this line as project is already defined; and without version we are getting error: 
```
CMake Error at /usr/share/cmake-3.24/Modules/WriteBasicConfigVersionFile.cmake:43 (message):
  No VERSION specified for WRITE_BASIC_CONFIG_VERSION_FILE()
Call Stack (most recent call first):
  /usr/share/cmake-3.24/Modules/CMakePackageConfigHelpers.cmake:237 (write_basic_config_version_file)
  CMakeLists.txt:706 (write_basic_package_version_file)
```